content/usgs-packages/sbtools_Intro.Rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "sbtools - Introduction"
+date: "9999-08-01"
+author: "Lindsay R. Carr"
+slug: "sbtools-intro"
+image: "img/main/intro-icons-300px/r-logo.png"
+output: USGSmarkdowntemplates::hugoTraining
+parent: Introduction to USGS R Packages
+weight: 2
+draft: true
+---
+
+```{r setup, include=FALSE, warning=FALSE, message=FALSE}
+library(knitr)
+
+knit_hooks$set(plot=function(x, options) {
+  sprintf("<img src='../%s%s-%d.%s'/ title='%s'/>", 
+          options$fig.path, options$label, options$fig.cur, options$fig.ext, options$fig.cap)
+
+})
+
+opts_chunk$set(
+  echo=TRUE,
+  fig.path="static/sbtools-intro/",
+  fig.width = 6,
+  fig.height = 6,
+  fig.cap = "TODO"
+)
+
+set.seed(1)
+```
+
+## Lesson Summary
+
+`sbtools` is an R package that enables access to the USGS web storage platform, [ScienceBase](https://www.sciencebase.gov). In this lesson, will teach how to interact with ScienceBase from the R console in order to create reproducible workflows.
+
+ScienceBase is collaborative scientific data and information management platform used by USGS scientists and their collaborators. Account holders can upload data using standard metadata practices, and choose to share the content publically or keep it private. An account is not needed to download publically available data, so the platform is a great way to share data released with publications.
```

Comment:
gotcha, their website seems like exactly the right starting point. I think maybe what they mean is that you _can_ include metadata with your data items, and that they support it by including a View button - for example,  has a file called NHDV2_PHYSIO_CONUS.xml that can be Viewed with some useful formatting. However, most of our powell center data items currently lack metadata, and that's also fine (much more feasible for temporary/preliminary/intermediate data artifacts), e.g., https://www.sciencebase.gov/catalog/item/5716b140e4b0ef3b7ca999a8. which is why i'm still in favor of taking out that phrase, despite what their webpage says.

if you're feeling excited about metadata and reluctant to take out all mention of it, consider "Account holders can upload data and metadata, and..." - to avoid implying that attaching metadata is a requirement for using sciencebase, which could deter people who'd otherwise consider using sciencebase for collaborative purposes